cmd/minikube/cmd/start.go

```diff
@@ -250,6 +259,8 @@ func runStart(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	validateKicBaseVersion(starter.Runner)
```

Comment:
Is this supposed to do the check for just Kicbase, or ISO as well? Because right now it seems to run on both. If it does run both we should rename it, but if it's just for kicbase we should add some logic to prevent it from running for ISO.